R/downloadableReactTable.R

```diff
@@ -0,0 +1,229 @@
+# -------------------------------------------
+# -- Application Downloadable React Table Module --
+# -------------------------------------------
+
+
+#' downloadableReactTable module UI function
+#'
+#' Creates a custom high-functionality table paired with a linked downloadFile
+#' button.  The table has search and highlight functionality, infinite scrolling,
+#' sorting by columns and returns a reactive dataset of selected items.
+#'
+#' downloadFile button will be hidden if \code{downloadableReactTable} parameter \code{downloaddatafxn} or
+#' \code{downloadableReactTableUI} parameter \code{downloadtypes} is empty
+#'
+#' @param id character id for the object
+#' @param downloadtypes vector of values for data download types
+#' @param hovertext download button tooltip hover text
+#' @param contentHeight viewable height of the table (any valid css size value)
+#' @param singleSelect whether the table should only allow a single row to be
+#' selected at a time (FALSE by default allows multi-select).
+#'
+#' @return list of downloadFileButton UI and DT datatable
```

Comment:
This looks like a copy/paste and you have not reviewed carefully - for instance there is a DT referenced here.  You need to investigate what parts of this will and will not be supported and update this documentation carefully.

Reply:
updated